src/openMVG_Samples/features_repeatability/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ target_link_libraries(openMVG_sample_main_repeatability_dataset
   openMVG_system
   openMVG_multiview
   vlsift
-  ${POPSIFT_LIBRARIES}
+  PopSift::popsift # ${POPSIFT_LIBRARIES}
```

Comment:
uff, i keep forgetting that this part is still messy. @fabiencastan 
We should move all the SIFT_Decriber.hpp classes in `openMVG/features` instead of keeping them in `nonFree` and leave there only the vlfeat. 
Then if we really want to do things right we should add a switch to enable/disable build of non-free components (vlfeat) hence disabling all the SIFT_Describers.
Finally adding popsift as (optional) dependency of `openMVG_features` would fix the thing once for all.